src/components/filter/AvailableHotels.js

```diff
@@ -26,23 +47,33 @@ function AvailableHotels(props) {
   }
   
   return (
-      <>
-        <section className="available-hotels col-md-12" id="avalableHotels">
-            <h1 className="available-hotels-title section-title col-md-12">Available hotels</h1>
-            <Slider {...settings}>
-                {props.hotelData.map((item) => (
-                <HotelItem 
-                    key = {item.name}
-                    data = {item}/>)
-                )}
-            </Slider>
-        </section>
-      </>
-  );
+      
+    <>
+    <section className="available-hotels col-md-12" id="avalableHotels">
+        <h1 className="available-hotels-title section-title col-md-12">Available hotels</h1>
+        <Switch>
+        <Route exact patch = "https://fe-student-api.herokuapp.com/api/hotel/:id">
+        <Slider {...settings}>
+          
+            {props.hotelData.map((item, id) => (
+            <div 
+              key = {id}
+              onClick={handleProceed}>
+              <Link to={`/hotels/${item.id}`}> 
+            <HotelItem 
+                data = {item}/>
+                 </Link>
+                 </div>)
+            )}
```

Comment:
extra div nesting
![image](https://user-images.githubusercontent.com/20189313/149681968-e8bc9a17-8ac2-473c-a57c-62625e32634f.png)

Reply:
but the meaning is correct, the elements are just a link to the page, and already this page should send a request, receive hotel data and then render